core-bundle/contao/modules/ModuleBreadcrumb.php

```diff
@@ -83,7 +85,7 @@ protected function compile()
 			(
 				'isRoot'   => true,
 				'isActive' => false,
-				'href'     => (($objFirstPage !== null) ? $this->generateContentUrl($objFirstPage) : Environment::get('base')),
+				'href'     => (($objFirstPage !== null) ? $this->generateContentUrl($objFirstPage) : $request->getBasePath()),
```

Comment:
What if `$request` is `null`? `Environment::get('base')` falls back to the current URL in this case I think.

Reply:
`Environment::getRequest()` would use `new Request(server: $_SERVER)` in that instance (which is also deprecated).

Reply:
You still have to handle the case, otherwise there will be a fatal error 🤷‍♂️